system-int-test/src/cmr/system_int_test/system.clj

```diff
@@ -41,7 +41,7 @@
              ;; A map of the components (echo, elastic, db, and message queue) to whether they are
              ;; in-memory or external
              :component-type-map component-type-map}]
-    (transmit-config/system-with-connections sys [:echo-rest :search :access-control :urs])))
+    (transmit-config/system-with-connections sys [:echo-rest :search :access-control :urs :ingest])))
```

Comment:
Now you have me curious - how did system integration tests ever work? We've always needed to hit ingest endpoints for the tests. Why did you specifically have to add this for your tests?